.eslintrc.js

```diff
@@ -1,25 +1,42 @@
 module.exports = {
   extends: 'airbnb',
+  env: { browser: true },
   rules: {
+    // ESLint ----------------------------------
+    'arrow-parens': ['error', 'always'],
```

Comment:
I guess, but you are again choosing a different style than the rest of the other web projects. 

You can also write `var f = (arg => arg + 1);`